docs/source/examples/workspaces/openshift/PinFile

```diff
@@ -37,3 +37,6 @@ openshift-new:
     credentials:
       api_endpoint: example.com:8443/
       api_token: mytokentextrighthere
+      # alternative way when --creds-path is used 
+      #filename: openshift.yaml
+      #profile: test
```

Comment:
This doesn't look to have the new way listed. Is this intentional? I figured moving the main use case to the proper usage would be desired.

Reply:
@herlo I might have just changed the documentation and forgot the workspaces. 
I updated in latest commit .